src/meta/processors/jobMan/ListEdgeIndexStatusProcessor.cpp

```diff
@@ -52,7 +52,13 @@ void ListEdgeIndexStatusProcessor::process(const cpp2::ListIndexStatusReq& req)
     });
     std::unordered_map<std::string, cpp2::JobStatus> tmp;
     for (auto &jobDesc : jobs) {
-        tmp.emplace(jobDesc.get_paras()[0], jobDesc.get_status());
+        auto paras = jobDesc.get_paras();
+        if (paras.size() == 1) {
+            tmp.emplace(paras[0] + "_all_edge_indexes", jobDesc.get_status());
+            continue;
+        }
+        paras.pop_back();
+        tmp.emplace(folly::join(",", paras), jobDesc.get_status());
```

Comment:
In graph layer，I saw that null argument is allowed, meaning that all indexes. How to unify with the storage layer?
https://github.com/vesoft-inc/nebula-graph/pull/600，

Reply:
The same goes for Storage.You can see the RebuildJobExecutor.cpp

Reply:
> The same goes for Storage.You can see the RebuildJobExecutor.cpp

Got, We do that in code `RebuildIndexTask.cpp`. nice job.